### PR TITLE
trivial: tiny cleanups and fixes

### PR DESCRIPTION
**Pull request by oxij**
- [X] Running with these applied.

### CODE REVIEW

File: pkgs/applications/networking/browsers/firefox/common.nix [hunk 1]
```diff
@@ -18,7 +18,7 @@
 ## optional libraries
 
 , alsaSupport ? stdenv.isLinux, alsaLib
-, pulseaudioSupport ? true, libpulseaudio
+, pulseaudioSupport ? stdenv.isLinux, libpulseaudio
```
Comment (roberth) [inline pkgs/applications/networking/browsers/firefox/common.nix]: Commit message should be descriptive like "firefox: disable pulseaudio support on non-Linux by default"

File: nixos/release.nix [hunk 2]
```diff
@@ -108,7 +108,7 @@ let
     let
       configEvaled = import lib/eval-config.nix {
         inherit system;
-        modules = [ module versionModule ];
+        modules = [ configuration module versionModule ];
```
Comment (roberth) [inline nixos/release.nix]: Looks like `[ configuration module versionModule ]` should be factored out across the entire file.

**Review by ttuegel — COMMENTED** [review]
Qt changes look fine to me.

**Comment by oxij** [comment]
Also pushed a piece of #49862 here.

**Pull request merged**